src/test/java/net/smartcosmos/dao/relationships/converter/RelationshipCreateToRelationshipEntityConverterTest.java

```diff
@@ -0,0 +1,54 @@
+package net.smartcosmos.dao.relationships.converter;
+
+import org.junit.*;
+
+import net.smartcosmos.dao.relationships.domain.RelationshipEntity;
+import net.smartcosmos.dao.relationships.util.UuidUtil;
+import net.smartcosmos.dto.relationships.RelationshipCreate;
+import net.smartcosmos.dto.relationships.RelationshipReference;
+
+import static org.junit.Assert.*;
+
+/**
+ * RelationshipCreateToRelationshipEntityConverter Tester.
+ *
+ * @author tcross
+ * @version 1.0
+ * @since <pre>Jun 28, 2016</pre>
+ */
+public class RelationshipCreateToRelationshipEntityConverterTest {
+
+    @Before
+    public void before() throws Exception {
+    }
+
+    @After
+    public void after() throws Exception {
+    }
+
+    /**
+     * Method: convert(RelationshipCreate createRelationship)
+     */
+    @Test
+    public void testConvert() throws Exception {
+        final String TEST_SOURCE_URN = "urn:thing:uuid:" + UuidUtil.getNewUuidAsString();
```

Comment:
## Helpful Hints
This will work and there is nothing fundamentally wrong but I recommend using different values for the `source` and `target` side of things.  This way it is harder to confuse the data.  For example:
```
TEST_TARGET_URN = "urn:target:uuid:" + ...
TEST_SOURCE_URN = "urn:source:uuid:" + ...
TEST_TARGET_TYPE = "Target";
TEST_SOURCE_TYPE = Source";
```
I know the URNs might not be strictly valid but that is not what is being tested.

Also typos can be avoided using `String.join(":", "urn", "source", "uuid", UuidUtil.getNewUuidAsString());`